app/posts/find-teacher-training/2023-04-18-removing-school-placement-vacancies.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Removing school placement vacancies
+description: We removed the school placement vacancy status as this was misleading for candidates
+date: 2023-04-18
+tags:
+  - school placements
+  - vacancies
+related:
+  items:
+    - text: Updating course vacancies
+      href: /publish-teacher-training-courses/updating-course-vacancies/
+---
+
+When a provider publishes a course, they can list all the schools where a candidate can gain experience. Previously, this list also showed whether there was a vacancy at the school.
+
+The vacancy status was a simple ‘yes’ or ‘no’ and required the provider to keep the course current.
+
+We removed the vacancy status because:
+
+- candidates do not apply to a vacancy at a school
+- providers often do not know the number of vacancies at their partner schools until late in the recruitment cycle
+- the number of vacancies can change during the recruitment cycle because candidates may need to withdraw
+- we are providing false hope that candidates will be placed at certain schools
```

Comment:
I don't think this has much bearing, as it still in an ambiguous  state

We still showing a list of schools with or without the potential of being placed in the desired school

Reply:
@defong These are the reasons garnered from research why vacancies are problematic. Can you elaborate on what is an ambiguous state and how we might mitigate against the ambiguity?